scripts/pre-commit

```diff
@@ -92,6 +94,14 @@ do
       echo "File $file is not indented."
       nocontinue=1
     fi
+
+    $CHECK_MACROS $file > /dev/null 2>&1
```

Comment:
I would keep the output of the script since it tells us the line number where the macro was found.